github/snapcraft.py

```diff
@@ -1,37 +1,122 @@
-#!/usr/bin/env python2
+#!/usr/bin/python2
 
-import urllib2
+import requests
 import urlparse
 
+import re
 
-def get_plugin_count(github_repo_url):
-    count = {}
-    repo = _get_repo_from_url(github_repo_url)
-    snapcraft_yaml_file = _get_snapcraft_yaml_file(repo)
-    if snapcraft_yaml_file:
-        for line in snapcraft_yaml_file:
-            line = line.strip()
-            if line.startswith('plugin: '):
-                plugin = line[8:]
-                if plugin in count:
-                    count[plugin] = count[plugin] + 1
-                else:
-                    count[plugin] = 1
-        snapcraft_yaml_file.close()
-    return count
+_scriplets = ["prepare", "build", "install", "version-script"]
```

Comment:
This is a constant, so it should be _SCRIPTLETS.
For consistency, please use only single quotes for strings, instead of double quotes.